Mirror/Runtime/Transport/EpicOnlineTransport/EOSSDKComponent.cs

```diff
@@ -199,13 +237,15 @@ public class EOSSDKComponent : MonoBehaviour {
                 AndroidJavaClass EOS_SDK_JAVA = new AndroidJavaClass("com.epicgames.mobile.eossdk.EOSSDK");
                 EOS_SDK_JAVA.CallStatic("init", context);
             }
-            
+
             // Prevent multiple instances
+         
             if (instance != null) {
                 Destroy(gameObject);
                 return;
             }
             instance = this;
+            DontDestroyOnLoad(instance);
```

Comment:
I didn't need this and in fact don't want it. This should be optional or left out.